src/mesh/BoundingBox.cpp

```diff
@@ -108,58 +105,67 @@ int BoundingBox::getDimension() const
 
 const std::vector<double> BoundingBox::dataVector() const
 {
-  return _bounds;
+  std::vector<double> bounds;
+  bounds.insert(bounds.end(), {_boundMin[0], _boundMax[0], _boundMin[1], _boundMax[1]});
```

Comment:
I guess you can merge these two lines as the vector takes an initializer list anyway.

Reply:
We (@jeremiahbintang ) merge these two lines.
```
std::vector<double> bounds({_boundMin[0], _boundMax[0], _boundMin[1], _boundMax[1]});
```